lab-data/yocto-stm32/bootlin-lab-data/nunchuk/linux/0002-ARM-dts-stm32mp157a-dk1-add-nunchuk.patch

```diff
@@ -9,11 +9,11 @@ Signed-off-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
  1 file changed, 13 insertions(+)
 
 diff --git a/arch/arm/boot/dts/stm32mp157a-dk1.dts b/arch/arm/boot/dts/stm32mp157a-dk1.dts
-index e3a36d3cdb45..97df04c5b1a0 100644
+index 2b01a01d1..cd731cb7b 100644
 --- a/arch/arm/boot/dts/stm32mp157a-dk1.dts
 +++ b/arch/arm/boot/dts/stm32mp157a-dk1.dts
-@@ -448,6 +448,19 @@
- 	};
+@@ -460,6 +460,19 @@
+ 	status = "disabled";
```

Comment:
This is just a change in the context, since we patch a patch, so this is fine :)